packages/clay-core/src/vertical-nav/VerticalNav.tsx

```diff
@@ -90,6 +95,13 @@ type Props<T extends Record<string, any> | string> = {
 	 */
 	large?: boolean;
 
+	/**
+	 * Messages used for announcement to SR. Use this for internationalization.
+	 */
+	messages?: {
+		error: string;
+	};
+
```

Comment:
@ilzamcmed do you think having an API for this would be necessary? I think this is more of a developer use case so we don't need internationalization of the message.

Reply:
Thanks @matuzalemsteles! You're right, I'll remove that.